code/game/mecha/combat/gygax.dm

```diff
@@ -33,6 +33,16 @@
 	ME = new /obj/item/mecha_parts/mecha_equipment/weapon/ballistic/missile_rack/flashbang
 	ME.attach(src)
 
+/obj/mecha/combat/gygax/examine_more(mob/user)
+	. = ..()
+	. += "<i>A light, fast and cheap combat mech designed and produced by Shellguard Munitions. \
+	Originally developed as a mobile flanker for open combat scenarios, a substantial flaw in the mech's Leg Overdrive systems caused poor sales. \
+	When using this system, the legs of the Gygax are prone to overheating and damaging the rest of the machine, leading overzealous pilots to harm themselves more than the enemy</i>."
```

Comment:
```suggestion
	When using this system, the legs of the Gygax are prone to overheating and damaging the rest of the machine, leading overzealous pilots to harm themselves more than the enemy.</i>"
```
Probably better to keep with the rest of the style and include the punctuation within your style tags.